legacy/src/main/java/org/opensearch/sql/legacy/rewriter/matchtoterm/TermFieldScope.java

```diff
@@ -7,7 +7,6 @@
 package org.opensearch.sql.legacy.rewriter.matchtoterm;
 
 import java.util.*;
```

Comment:
Not related by this fix, but do you have `spotless` rule for star imports?

Reply:
What would you want to happen, would you want star imports to be converted to the correct imports?

Reply:
Yes, I think our checkstyle config rejects star imports